docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -64,6 +66,40 @@ For example:
 The same applies to `slotProps` and `componentsProps`.
 :::
 
+### Slot `shortcuts`: add new parameters to the `onChange` callback
+
+:::warning
+The following breaking change only impacts you if you are overriding the `shortcuts` slot to create your own custom UI.
+If you are just passing shortcuts to the default UI using `slotProps={{ shortcuts: [...] }}` then you can safely skip this section.
+:::
+
+The `onChange` callback fired when selecting a shortcut now requires two new parameters:
+
+- The [`changeImportance`](/x/react-date-pickers/shortcuts/#behavior-when-selecting-a-shortcut) of the shortcut.
+- The `item` containing the entire shortcut object.
```

Comment:
I assume not providing the `changeImportance` will lead to broken shortcuts.
But for the `item` I'm not sure about what's the impact.

If I correctly understand, the usage of `item` is to provide some context to the `onChange`. What do you think about saying the last parameter is the shortcut context? Such that devs know they can pass whatever they think valuable for the `onChange`

Reply:
> If I correctly understand, the usage of item is to provide some context to the onChange

Yes, it's just passed directly to `context.shortcut`.

> What do you think about saying the last parameter is the shortcut context? Such that devs know they can pass whatever they think valuable for the onChange

I guess it's unlikely that we will ever use this value for something else than just forwarding it to the context, so people should be able to pass whatever they want indeed.

Would you agree that it's not the role of the migration guide though? More the role of the shortcut doc if we want to highlight this ability at some point.

Reply:
Yes, it's just that it's the only place where this feature is explained for now